sonar-db/src/test/java/org/sonar/db/component/ComponentDbTester.java

```diff
@@ -32,71 +31,69 @@
 public class ComponentDbTester {
   private final DbTester db;
   private final DbClient dbClient;
-  private final DbSession dbSession;
 
   public ComponentDbTester(DbTester db) {
     this.db = db;
     this.dbClient = db.getDbClient();
-    this.dbSession = db.getSession();
   }
 
   public SnapshotDto insertProjectAndSnapshot(ComponentDto component) {
-    dbClient.componentDao().insert(dbSession, component);
```

Comment:
From what you explained to me, this commit "Do not keep reference on DbSession in ComponentDbTester" could be removed